skimage/feature/_texture.pyx

```diff
@@ -264,3 +265,89 @@ def _local_binary_pattern(double[:, ::1] image,
                 output[r, c] = lbp
 
     return np.asarray(output)
+
+
+# Constant values that are used by `multiblock_local_binary_pattern` function.
+# Values represent offsets of neighbour rectangles relative to central one.
+# It has order starting from top left and going clockwise.
+cdef:
+    Py_ssize_t[::1] mlbp_x_offsets = np.asarray([-1, 0, 1, 1, 1, 0, -1, -1])
+    Py_ssize_t[::1] mlbp_y_offsets = np.asarray([-1, -1, -1, 0, 1, 1, 1, 0])
+
+def _multiblock_lbp(float[:, ::1] int_image,
+                    Py_ssize_t x,
+                    Py_ssize_t y,
+                    Py_ssize_t width,
+                    Py_ssize_t height):
+    """Multi-block local binary pattern.
```

Comment:
I'd favor ending the first line with (MB-LBP) as the acronym isn't officially defined elsewhere, and we have the room in the line.